scripts/qa_crowbarsetup.sh

```diff
@@ -2674,6 +2674,10 @@ function custom_configuration
                         proposal_set_value neutron default "['attributes']['neutron']['ml2_type_drivers']" "['gre','vxlan','vlan']"
                         if [[ $want_dvr = 1 ]]; then
                             proposal_set_value neutron default "['attributes']['neutron']['use_dvr']" "true"
+                            # DVR with GRE or VXLAN requires L2 population
+                            if [[ $networkingmode = vxlan ]] || [[ $networkingmode = gre ]]; then
```

Comment:
I think that in shell is done like `[[ $networkingmode = "vxlan" || $networkingmode = "gre" ]];`